README.md

```diff
@@ -32,13 +32,22 @@ A modern zim fileserver which can handle multiple zim files by serving a beautif
  
 
 # Installation
-- Install go and compile it using `go build -o zimserver`
+- Install go and compile it using `go build`
 or
 - Download the latest release
 
+# Configuration
+### Example
+config.toml:
+```toml
+[Config]
+  LibraryPath = "./library"
+  Port = ":8080"
+```
+
 # Usage
-Create a folder `library` and place your .zim files inside it, or link them using symlinks.<br>
-Run the binary and go to `https://localhost:8080`
+Your `LibraryPath` must contain your .zim files inside it.  
```

Comment:
Why did you remove the `symlink` hint?

Reply:
Sorry, I forgot to put this part in. :(
Which is a bit of a pity, since it is a feature that is presented further down in the README...
This is fixed in c2dbfac.